packages/components/pages/f-contact-preferences/package.json

```diff
@@ -48,16 +48,19 @@
   ],
   "dependencies": {
     "@justeat/f-services": "1.7.0",
-    "@justeat/f-globalisation": "1.0.0"
+    "@justeat/f-globalisation": "1.0.0",
+    "@justeat/f-vue-icons": "3.3.0"
```

Comment:
Could this be a `devDependency` potentially? I imagine the icons get built into the package at build time (but could be wrong)